providers/Spotify/mod.ts

```diff
@@ -173,8 +173,12 @@ export class SpotifyReleaseLookup extends ReleaseApiLookup<SpotifyProvider, Albu
 					this.constructReleaseApiUrl(),
 					this.options.snapshotMaxTimestamp,
 				);
-				if (cacheEntry.content?.albums?.items?.length) {
-					return cacheEntry.content.albums.items[0].id;
+				const releases = cacheEntry.content?.albums?.items;
+				if (releases?.length) {
+					if (releases.length > 1) {
+						this.warnMultipleResults(releases.toSpliced(0, 1).map((release) => release.external_urls?.spotify));
```

Comment:
```suggestion
						this.warnMultipleResults(releases.slice(1).map((release) => release.external_urls?.spotify));
```
I had to lookup what `toSpliced` is actually doing, it is a quite new method which does the same as a simple `slice` can do in this case.